app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -188,10 +188,17 @@ def async_state_done(current_async_state)
         },
       )
 
-      form_response.extra[:ssn_is_unique] = DuplicateSsnFinder.new(
-        ssn: idv_session.ssn,
-        user: current_user,
-      ).ssn_is_unique?
+      if current_async_state.result[:ssn_is_unique]
+        # new state
+        # It would be cleaner to reverse this, but what about existing logs?
+        form_response.extra[:ssn_is_unique] = current_async_state.result[:ssn_is_unique]
+      else
+        # old state
```

Comment:
Handling the 50/50 state, and anything in flight. When this goes out, ResolutionProofingJob will start storing `ssn_is_unique` on the result going forward. In the near-term, if it's not set, fall back to the old pattern.

After this has been out for a short bit a followup PR can remove this conditional.

Reply:
An alternative: You can leave this code the same and always override the value from the job. Then when the new code is in place in the background job remove this code and stop overwriting what the job does.